<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>726</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>726</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>20</y>
      <width>992</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainLayer">
     <item row="7" column="0" colspan="3">
      <widget class="QWidget" name="wgtHBBar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>OPEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblfilename">
         <property name="minimumSize">
          <size>
           <width>188</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Videofile: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>341</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConfig">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>CONFIG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnProcess">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>PROCESS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="dspLayer">
       <item>
        <widget class="QVideoWidget" name="mediaPlayerWidget" native="true"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QWidget" name="wgtHTBar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="uiLayer">
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBackward">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>BACKWARD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>PLAY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForward">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>FORWARD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2" rowspan="3">
      <widget class="QWidget" name="wgtVRBar" native="true">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="3">
      <widget class="QWidget" name="wgtVLBar" native="true">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="sldvideopos">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblvideolen">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>--:--</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
